test/functional/testlib.bash

```diff
@@ -2030,6 +2085,8 @@ add_to_pack() {
 		sudo tar -C "$item_path" -rf "$version_path"/pack-"$bundle"-from-"$version".tar --transform "s,^,staged/," "$(basename "$item")"
 	elif [[ "$item" = *"/delta"* ]]; then
 		sudo tar -C "$version_path" -rf "$version_path"/pack-"$bundle"-from-"$version".tar delta/"$(basename "$item")"
+	elif [[ "$item" = *"/Manifest."*".D"* ]]; then
+		sudo tar -C "$version_path" -rf "$version_path"/pack-"$bundle"-from-"$version".tar "$(basename "$version_path")"/"$(basename "$item")"
```

Comment:
It looks like you accidentally swapped $(dirname "$item") with $version_path instead of item_path.

sudo tar -C "$version_path"...... should be sudo tar -C "$item_path".....

Reply:
I changed that path to $version_path un purpose.

Before, the item being passed here was the delta manifest in the version directory, e.g.  <environment>/web-dir/30/Manifest.<bundle>.D.20
so $item_path was resolving to <environment>/web-dir/30 and $version_path was therefore resolving to <environment>/web-dir, so item_path was the actual version_path for that one file (I know it is confusing but it was different than the other files).

Now, after the last update, the item being passed to the function is the delta manifest in the TO directory, e.g. <environment>/web-dir/30/30/Manifest.<bundle>.D.20, this way when we add it to the pack, the Manifest goes inside the version directory like 30/Manifest.<bundle>.D.20, but because of this, now $item_path is <environment>/web-dir/30/30/ and $version_path is actually correct <environment>/web-dir/30.

I hope the explanation didn't confuse you more :P

Reply:
I see now... You had more changes than I thought and I needed more updates to my test branch. Looks good!